exercise_060_cluster_base/src/main/scala/akka/cluster/pi/ClusterStatusTracker.scala

```diff
@@ -0,0 +1,128 @@
+package akka.cluster.pi
+
+/**
+ * Copyright © 2016-2019 Lightbend, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * NO COMMERCIAL SUPPORT OR ANY OTHER FORM OF SUPPORT IS OFFERED ON
+ * THIS SOFTWARE BY LIGHTBEND, Inc.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * ClusterStatusTracker is an actor that registers for Akka Cluster
+ * Membership, Reachability, and Leadership Events and translates this
+ * to 'Visualiser' events.
+ *
+ * In its current form, the processing of Akka Cluster events
+ * and the visualisation of the cluster status are in separate components.
+ *
+ * Next step is to make the `ClusterStatusTracker` track the state of the
+ * cluster and to allow more than one, normally of different type, visualisers
+ * to register.
+ */
+
+import akka.actor.typed.scaladsl.{ActorContext, Behaviors, TimerScheduler}
+import akka.actor.typed.{ActorRef, Behavior}
+import akka.cluster.ClusterEvent._
+import akka.cluster.Member
+import akka.cluster.MemberStatus.{Up, WeaklyUp}
+import akka.cluster.typed.{Cluster, Subscribe}
+
+import scala.concurrent.duration._
+
+object ClusterStatusTracker {
+
+  sealed trait Event
```

Comment:
'event' might cause confusion with persistent events - so `Command` might be slightly nicer?

Reply:
@raboof 
I'm hesitating between `Command` and `ClusterEvent`. WDYT?

Reply:
I like `Command` as the scope is clear from context and `ClusterEvent` could be confused with `akka.cluster.ClusterEvent`, but no strong feelings here

Reply:
> `ClusterEvent` could be confused with `akka.cluster.ClusterEvent`

True... I had changed it to `ClusterEvent` already. `Command` is a bit generic and hides the fact that these messages are `ClusterEvent`'s. Not sure what to choose... naming is hard.

Reply:
May go for a mix of the class and 'event' such as `ClusterStatusTrackerEvent`?

Reply:
@eloots did you consider having this class part of common module? I guess it's commonly used across excercies